src/main/kotlin/apply/application/MyMissionService.kt

```diff
@@ -94,4 +96,23 @@ class MyMissionService(
             judgmentRecord = judgment?.lastRecord
         )
     }
+
+    fun findByMemberIdAndMissionId(memberId: Long, missionId: Long): MyMissionResponse {
+        val mission = missionRepository.getOrThrow(missionId)
+
+        evaluationTargetRepository.findByEvaluationIdAndMemberId(mission.evaluationId, memberId)
+            ?: throw NoSuchElementException("과제가 존재하지 않습니다. id: $missionId")
+
+        if (!mission.isDescriptionViewable) {
+            throw NoSuchElementException("과제가 존재하지 않습니다. id: $missionId")
+        }
+
+        val assignment = assignmentRepository.findByMemberIdAndMissionId(memberId, missionId)
+
+        return MyMissionResponse(
+            mission = mission,
+            submitted = assignment != null,
+            markdownToEmbeddedHtml(mission.description),
+        )
```

Comment:
c: 아래 코드는 어떨까요? 조건문에 중괄호를 추가하거나, 생성자를 호출할 때 인자에 줄 바꿈을 추가할 수도 있습니다.

```suggestion
        val mission = missionRepository.getOrThrow(memberId)
        val evaluationTarget = evaluationTargetRepository.findByEvaluationIdAndMemberId(mission.evaluationId, memberId)
        if (!mission.isDescriptionViewable || evaluationTarget == null) throw NoSuchElementException("과제가 존재하지 않습니다. id: $missionId")
        val assignment = assignmentRepository.findByMemberIdAndMissionId(memberId, missionId)
        return MyMissionResponse(mission, markdownToEmbeddedHtml(mission.description), submitted = assignment != null)
```